test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -161,7 +161,7 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [Fact]
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.NotNetCore30))] // netcore3.0 output differs from Baseline
```

Comment:
Disabling these tests on `netcoreapp3.0` feels wrong. What happens when we completely move to .NET Core 3.0? We will then have to spend effort on re-enabling these tests for 3.0.

Is there a way we can investigate why the baselines differ? Rather than just shutting the tests off?